Source/WebInspectorUI/UserInterface/Models/FontStyles.js

```diff
@@ -138,6 +138,10 @@ WI.FontStyles = class FontStyles
 
         const createIfMissing = true;
         let cssProperty = this._effectiveWritablePropertyForName(targetPropertyName, createIfMissing);
+
+        if (this._propertiesMap.get(targetPropertyName)?.isImportant)
+            targetPropertyValue += " !important";
```

Comment:
NIT: are we 100% sure that `targetPropertyValue` can't already have `!important` in it?

Reply:
The `value` that ends up here is the value of the `<input type="range">` elements in the Fonts sidebar. 
It gets there from `WI.CSSProperty.value` which is guaranteed to strip `!important`:

https://github.com/WebKit/WebKit/blob/aef2df021f711fd6b5e073d41fac755bb607dbcb/Source/WebInspectorUI/UserInterface/Models/CSSProperty.js#L370-L376

`targetPropertyValue` is either that `value` as string or a transformed string according to this mapping:
https://github.com/WebKit/WebKit/blob/aef2df021f711fd6b5e073d41fac755bb607dbcb/Source/WebInspectorUI/UserInterface/Models/FontStyles.js#L73-L85

There's no `!important` ending up here. That's why we need to append `!important` ourselves when writing back to the stylesheet.